astro/src/content/docs/lifecycle/migrate-users/genericmigration.mdx

```diff
@@ -0,0 +1,275 @@
+
+If you have errors logging in, you can use the FusionAuth database directly to see if your users were imported, and check their hashes manually.
+
+You can use any PostgreSQL browser. DBeaver will work. The connection details are in the files `docker-compose.yml` and `.env` in the `fusionauth-import-scripts/generic/fusionAuthDockerFiles/` directory.
+
+In your database IDE, create a new PostgreSQL connection with the following details:
+
+- <InlineField>URL:</InlineField> `jdbc:postgresql://localhost:5432/fusionauth`
+- <InlineField>Host:</InlineField> `localhost`
+- <InlineField>Port:</InlineField> `5432`
+- <InlineField>Database:</InlineField> `fusionauth`
+- <InlineField>Username:</InlineField> `fusionauth`
+- <InlineField>Password:</InlineField> `hkaLBM3RVnyYeYeqE3WI1w2e4Avpy0Wd5O3s3`
+
+Log in to the database and browse to `Databases/fusionauth/Schemas/public/Tables`. The `identities` and `users` tables will show the login credentials and user personal information.
+
+## Use FusionAuth As Your Authentication Provider
+
+Now that your users have been migrated into FusionAuth, how do you authenticate them in your app?
+
+The first step is to set your OAuth callback URL in the [FusionAuth admin UI](http://localhost:9011/admin). Under **Applications** edit your `Test` application and set the <InlineField>Authorized redirect URLs</InlineField> to `http://localhost:7771/callback`.
+
+<img src={`/img/docs/lifecycle/migrate-users/provider-specific/${migration_source_dir}/authorised-redirect.png`} alt="Authorised redirect URL." width="1200" role="bottom-cropped" />
+
+Now run the command below in the interactive Docker terminal to see the original Express app rewritten to use FusionAuth for authentication.
+
+```sh
+node 7_webAppWithFa.mjs
+```
+
+Browse to `http://localhost:7771`. You'll see that the sign-in page has been replaced by FusionAuth. You can style this page however you like. For more information, see the full [quickstart guide for Express](/docs/quickstarts/quickstart-javascript-express-web).
+
+The new application code looks very similar to the original, except that the login and hashing code has been replaced by OAuth 2.0 calls to FusionAuth via the Node.js [Passport library](https://www.npmjs.com/package/passport).
+
+<Aside type='caution'>
+ Note that the `authOptions` object defined at the bottom stores secrets directly in the code. In reality, you should move these to a `.env` file that is not checked into GitHub. Be sure to use `host.docker.internal` instead of `localhost` for URLs that call a server directly (browser URLs still use `localhost`).
+</Aside>
+
+### Delete The Docker Containers
+
+Push Ctrl + C in all terminals to stop the Docker instances. Run the code below on your host machine to remove the Docker containers and images if you are done testing.
+
+```sh
+docker rm app app_db fa fa_db
+docker rmi postgres:16.2-alpine3.19 node:alpine3.19 fusionauth/fusionauth-app:1.48.3 postgres:16.0-bookworm
```

Comment:
because of the change here, I think the `rmi fusionauth/fusionauth-app:1.48.3` might fail based on latest.

https://github.com/FusionAuth/fusionauth-import-scripts/pull/21#discussion_r1750594196

Reply:
fixed